customizations/generator/openapi/src/main/kotlin/com/expediagroup/sdk/generators/openapi/OpenApiSdkGenerator.kt

```diff
@@ -100,7 +100,7 @@ class OpenApiSdkGenerator {
                     addGlobalProperty(CodegenConstants.MODELS, "")
                     addGlobalProperty(CodegenConstants.MODEL_DOCS, "false")
 
-                    supportingFiles.add("${namespace}Client.kt")
+                    supportingFiles.add("${namespace.replaceFirstChar(Char::titlecase)}Client.kt")
```

Comment:
Maybe we should use the previously used `pascalCase` method since it's safer than an AdHoc approach?